server/routerlicious/kubernetes/routerlicious/templates/fluid-configmap.yaml

```diff
@@ -68,14 +68,23 @@ data:
             "numberOfMessagesPerTrace": {{ .Values.alfred.numberOfMessagesPerTrace }},
             "sessionStickinessDurationMs": {{ .Values.alfred.sessionStickinessDurationMs }},
             "throttling": {
-                "restCalls": {
-                    "maxPerMs": {{ .Values.alfred.throttling.restCalls.maxPerMs }},
-                    "maxBurst": {{ .Values.alfred.throttling.restCalls.maxBurst }},
-                    "minCooldownIntervalInMs": {{ .Values.alfred.throttling.restCalls.minCooldownIntervalInMs }},
-                    "minThrottleIntervalInMs": {{ .Values.alfred.throttling.restCalls.minThrottleIntervalInMs }},
-                    "maxInMemoryCacheSize": {{ .Values.alfred.throttling.restCalls.maxInMemoryCacheSize }},
-                    "maxInMemoryCacheAgeInMs": {{ .Values.alfred.throttling.restCalls.maxInMemoryCacheAgeInMs }},
-                    "enableEnhancedTelemetry": {{ .Values.alfred.throttling.restCalls.enableEnhancedTelemetry }}
+                "restTenantCalls": {
```

Comment:
I wonder if it would be better to use perTenant and perCluster terminology everywhere to describe its usage better.

Reply:
Add perTenant and perCluster as the postfix, so that we can apply the same in the other throttler if in need.  :)